.github/workflows/ci.yml

```diff
@@ -293,4 +293,5 @@ jobs:
           docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
           docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
           docker-images: >-
-            springcloudtask/timestamp-batch-task:2.0.0
\ No newline at end of file
+            springcloudtask/timestamp-batch-task:2.0.1
+            
```

Comment:
LOL. The trailing whitespace makes it think its a line and still needs a newline. Can you chop out the trailing whitespace?